style.css

```diff
@@ -478,7 +475,178 @@ textarea:focus {
   font-size: 32px;
 }
 
+.pop-up {
+  display: none;
+}
```

Comment:
> - [ ] [OPTIONAL] Also, it would be great to make the popup look good on large screens. Right now, when you open the portfolio on a large screen, the popup does not show completely, so it can lead to a bad user experience.

- As required by the previous reviewer, I strongly suggest you make it so the cards are displayed correctly,  You can use the `overflow: scroll;` CSS property for that.